runner/runner.go

```diff
@@ -49,7 +50,7 @@ func (r *Runner) Run(ctx context.Context, query ...string) error {
 		return errors.New("no keys provided")
 	}
 
-	var censysRateLimiter, fofaRateLimiter, shodanRateLimiter, shodanIdbRateLimiter, quakeRatelimiter, hunterRatelimiter, zoomeyeRatelimiter, netlasRatelimiter, criminalipRatelimiter *ratelimit.Limiter
+	var censysRateLimiter, fofaRateLimiter, shodanRateLimiter, shodanIdbRateLimiter, quakeRatelimiter, hunterRatelimiter, zoomeyeRatelimiter, netlasRatelimiter, criminalipRatelimiter, publicwwwRatelimiter *ratelimit.Limiter
```

Comment:
@xm1k3 , I think we can replace this logic of creating new ratelimiter and passing it to every source.

I think we can use https://github.com/projectdiscovery/ratelimit/blob/main/keyratelimit.go instance in `type Session struct` to apply ratelimit based on key(aka source)